doc/user/content/concepts/views.md

```diff
@@ -15,104 +15,119 @@ aliases:
+memory** within that [cluster](/concepts/clusters/). The in-memory up-to-date
+results are accessible to queries within the cluster, even for queries that do
+not use the index key(s).
+
+See also:
 
-Indexes are local to a cluster.  Queries in a different cluster cannot use the
-indexes in another cluster.
+- [Indexes](/concepts/indexes)
+- [Optimization](/transform-data/optimization)
+- [`CREATE INDEX`](/sql/create-index/)  for complete syntax information
 
 ## Materialized views
 
-A materialized view, like a non-materialized view, saves a query under a name to
-provide a shorthand for referencing the query. But, unlike a non-materialized
-view, the query is executed during the view creation, and a materialized view
-**persists and incrementally updates** its results in durable storage.
+In Materialize, a materialized view is a view whose underlying query is executed
+during the view creation, and the view results are both **persisted and
+incrementally updated** in durable storage. Materialized views can be referenced across [clusters](/concepts/clusters/).
+
+```mzsql
+CREATE MATERIALIZED VIEW my_mat_view_name AS
+  SELECT ... FROM ...  ;
+```
 
 You can index a materialized view to maintain the results in memory
 within the cluster. This enables queries within the cluster to use the index to
-access view results from memory.  See [Indexes and materialized views](#indexes-and-materialized-views)
-for more information.
+access view results from memory.
+
 
 See also:
 
-- [`CREATE MATERIALIZED VIEW`](/sql/create-materialized-view)
+- [`CREATE MATERIALIZED VIEW`](/sql/create-materialized-view) for complete
+  syntax information
 
 
 ### Indexes and materialized views
 
-Indexes can improve query performance. In Materialize,
-[indexing](/concepts/indexes/) a materialized view maintains the results in
-memory within the [cluster](/concepts/clusters/).  Because materialized views
-maintain the up-to-date results in durable storage, indexes on materialized
-views serve up-to-date results without themselves performing the incremental
-computation. The in-memory up-to-date results are accessible to queries within
-the cluster, even for queries that do not use the index key(s).
+In Materalize, materialized views can be indexed.
+
+```mzsql
+CREATE INDEX idx_on_my_view ON my_mat_view_name(...) ;
+```
 
-Indexes are local to a cluster.
+Because materialized views maintain the up-to-date results in durable storage,
+indexes on materialized views serve up-to-date results without themselves
+performing the incremental computation. The in-memory up-to-date results are
+accessible to queries within the cluster, even for queries that do not use the
+index key(s).
 
 See also:
 
-- [Indexes](/concepts/indexes/)
-- [`CREATE INDEX`](/sql/create-index/)
+- [Indexes](/concepts/indexes)
+- [Optimization](/transform-data/optimization)
+- [`CREATE INDEX`](/sql/create-index/)  for complete syntax information
+
+
+## General information
```

Comment:
Should this be a "TL;DR" section at the beginning of the page, rather than the end?

Reply:
So, this is more in prep for a takeaway summary page in the future.  But, on this page, I think it makes sense at the end.